src/App.js

```diff
@@ -21,31 +21,34 @@ function App() {
   });
 
   useEffect(() => {
-    const userId = getLoggedInUserId();
-    if (userId === null) {
-      return;
-    }
-    client
-      .get(`/user/${userId}`)
-      .then(res => setUser(res.data.data.user))
-      .catch(err => console.log(err));
+    const userId = getLoggedInUserId()
+    if (userId === null) { return }
+
+    fetch(`${process.env.REACT_APP_API_URL}/user/${userId}`, {
```

Comment:
What's the reasoning behind change the existing `client` call to a `fetch`? It's fine if there's a good reason for it, if there is you may also want to add a `.catch` to the chain to handle promise rejections